### DISCUSSION TITLE
WHY CAN T I CONNECT MY DATABASE [Jul 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6130] [Discord Threads]
> 🙋‍♂️ **deniz.sh**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### I hope you are using the edit connection JSON option ?  
#  
> 👨‍💻 **pranavxc**  
#### And providing client value as `"Mysql"`  
#  
> 👨‍💻 **pranavxc**  
#### If the above 2 assumptions are right then change it with `"mysql2"` and it will work  
#  
> 👨‍💻 **pranavxc**  
####   
![image0](https://cdn.discordapp.com/attachments/1123337606785138709/1123675397410787358/Screenshot_2023-06-28_at_11.34.41_PM.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1123337606785138709)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
